terraform/modules/lambda/ccs-virus-scan/lambda_function.py

```diff
@@ -0,0 +1,20 @@
+import urllib.parse as urlparse
+import urllib.request as request
+import json
+import os
+
+
+def lambda_handler(event, context):
+    host = os.environ['HOST']
+    for record in event["Records"]:
+        key    = record["s3"]["object"]["key"]
+        bucket = record["s3"]["bucket"]["name"]
+        url    = host + ":4567/scan?" + urlparse.urlencode({"key": key, "bucket": bucket})
```

Comment:
Might be tidier to keep the port at the infra level wit the rest of the base URL.